lab/instant.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "lab",
+  "name": "Labaratory Package",
```

Comment:
Should we name it `Laboratory Workflows Package`? also, notice the typo in `Laboratory`.